fitnessllm_dataplatform/stream/strava/services/etl_interface.py

```
@@ -0,0 +1,265 @@
+                    ),
+                )
+                result = job.result()
+            if result.state == 'DONE':
+                self.insert_metrics(
+                    metrics_list=metrics,
+                    destination=f"{self.client.project}.dev_metrics.metrics",
+                    timestamp=result.created,
+                    status=Status.SUCCESS,
+                )
+                return
+            raise Exception("Job did not complete successfully")
```

Ruff (0.8.2)

Abstract `raise` to an inner function

(TRY301)